src/controllers/itemdetailpage.js

```diff
@@ -418,13 +426,17 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
         var offset = parentNameLast ? ".25em" : ".5em";
 
         if (html && !parentNameLast) {
-            html += '<h3 class="itemName infoText" style="margin: .25em 0 .5em;">' + name + '</h3>';
+            if (!layoutManager.mobile && parentNameHtml[1]) {
+                html += '<h3 class="itemName infoText" style="margin: .25em 0 .5em; font-weight: 600;">' + parentNameHtml[1] + ' - ' + name + '</h3>';
```

Comment:
I'd rather we have font weights specified via css classes, not inline

Reply:
I agree, avoid inline CSS in all cases. It's a mess to maintain.

Reply:
@JustAMan @MrTimscampi I tried writing the rules in the ``itemName`` and ``infoText`` classes but for some reason they weren't applied, even when using ``!important``, so I thought that it wasn't that important. I'm not sure though if that was caused by gulp not reloading the page correctly, so let me check that. But I suspect that this might need its own class or some kind of refactoring

Reply:
@JustAMan @MrTimscampi Seems that it was indeed my browser's view that didn't reload the preview in real time. Fixed in my new commit.